coretex/entities/task_run/parameter/parameters/list_enum_parameter.py

```diff
@@ -53,3 +55,12 @@ def parseValue(self, type_: ProjectType) -> Optional[Any]:
             return None
 
         return [options[value] for value in selected]
+
+    def overrideValue(self, value: Optional[Any]) -> Optional[Any]:
+        if value is None:
+            return None
+
+        try:
+            return json.loads(value.replace("'", "\""))
```

Comment:
Enum list parameter value should be a list of strings, why is it being treated as a json object?

Reply:
The command line input will be gotten as a string ("['str1', 'str2', 'str3']"), which is most effectively cast into the corresponding json like object (list), using json.loads

Reply:
`tap` (Typed Argument Parser) supports lists as parameters, why do we have to manually parse them?

Reply:
Lists are parsed in a way that does not work for our context.
For example a list of strings will get parsed like this:
--parameter one two    ->    [["o", "n", "e"], ["t", "w", "o"]]

Reply:
This is not true, `--parameterName first second third` gets parsed into `["first", "second", "third"]`

Reply:
That is not what I encountered. The desired results were only gotten by using nargs = "+" (meaning capture 1 and more passed args) with type = {non_list_type} (like int)